src/Pact/Types/Gas.hs

```diff
@@ -189,6 +189,8 @@ data GasArgs
   | GHyperlaneDecodeTokenMessage !Int
   -- ^ Cost of hyperlane-decode-token-message on this size (in bytes) of the
   --   hyperlane TokenMessage base64-encoded string.
+  | GKeccak256 !Int
```

Comment:
re: previous comment, I believe this should be `[Int]` (with a base charge amount if the list is empty)